.gitpod.yml

```diff
@@ -0,0 +1,29 @@
+tasks:
+  - init: |
+      # remove once latest maven works again or we switch to maven wrapper
+      sed -i 's/sdkman_auto_answer=false/sdkman_auto_answer=true/' $HOME/.sdkman/etc/config
+      sdk selfupdate
+      sdk update
+      sdk install maven 3.8.4
+      sdk use maven 3.8.4
+
+      brew install gh
+
+      mvn -am -pl war,bom -P quick-build install
+    command: |      
+      cd war
+      curl https://gist.githubusercontent.com/timja/fe73f0a0f98afe4d9356e7ce6752d95b/raw/98ad71080e91ed75824ec410ff905419e3788065/init.sh | bash
```

Comment:
I wonder whether this line is really needed? I was able to get UI without it too and redirection did not work anyway.

How `GIT_POD_URL` is populated in this file?

Reply:
let me try without it, it helped before I found out about the `host=0.0.0.0` thing.

Reply:
Ah I had forgotten to commit the host thing.

Redirects should work now.

If I don't add this file then you get this warning:

![image](https://user-images.githubusercontent.com/21194782/169481513-1984f668-7656-4925-9db9-3800fa2eddc5.png)

Which means certain things wouldn't work but generally it's fine.

So could omit it but probably doesn't do any harm.

----

an alternative may be to install the configuration as code plugin in the init script which would allow more config to be done if people wanted.

----

> How `GIT_POD_URL` is populated in this file?

by running `gp url 8080`